core/experiment-2/packages/admin/pages/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FC, lazy, useState } from "react";
+import Button from "@mui/material/Button";
+import Typography from "@mui/material/Typography";
+import Box from "@mui/material/Box";
+import Tooltip from "@mui/material/Tooltip";
+import Plugin1 from "plugins/Plugin1";
+import Plugin2 from "plugins/Plugin2";
```

Comment:
Maybe I'm exceeding the purpose of this experiment. The big challenge here will be that we won't necessarily have the Plugin1 and Plugin2 at the build time.
The great question is: can we make the plugins be
- [ ] dynamically loaded at run time
- [ ] and loaded at the server side

at the same time? It may be a matter for another experiment. I don't know.

Reply:
Let me think about it

Reply:
I use injectScript to inject remote at runtime and load plugins which can be used on server and client

Reply:
I already merged it! Thanks. 

Just a follow up question. I still see the plugin being loaded at the Network tab of Chrome DevTools. Do you know how we can load it as SSR only?